index.html

```diff
@@ -839,7 +839,7 @@ <h2>
       </section>
       <section data-link-for="OrientationType">
         <h2>
-          <dfn>Locking orientation algorithm</dfn>
+          <dfn>Applying an orientation lock algorithm</dfn>
```

Comment:
Try to avoid double defining, otherwise it's not clear which one to link to. 

```suggestion
          Apply an orientation lock algorithm
```

Similarly below...